### PR TITLE
Update Japanese Translation (Line 960-)

### PR DESCRIPTION
**Pull request by yosshy**
/hold for japanese developer's review
/assign @atoato88
/cc @YuikoTakada @t-inu @s-kawamura-w664

### CODE REVIEW

**Comment by yosshy** [comment]
I signed it

File: translations/ja.json [hunk 1]
```diff
@@ -954,64 +954,64 @@
...
-	"unable to bind flags": "フラグをバインドすることができませんでした",
-	"unable to daemonize: {{.err}}": "",
-	"unable to delete minikube config folder": "minikube の設定フォルダーを削除できませんでした",
-	"unable to set logtostderr": "logtostderr を設定することができませんでした",
-	"unpause Kubernetes": "Kubernetes を再開させます",
-	"unset failed": "取り消しが失敗しました",
-	"unsets PROPERTY_NAME from the minikube config file.  Can be overwritten by flags or environmental variables": "minikube の設定ファイルから PROPERTY_NAME の値を取り消します。フラグ、あるいは環境変数で上書き可能です",
-	"unsets an individual value in a minikube config file": "minikube の設定ファイルの個々の値を取り消します",
-	"unsupported or missing driver: {{.name}}": "サポートしていない、あるいは不足しているドライバーです: {{.name}}",
+	"toom any arguments ({{.ArgCount}}).\\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE": "引数 ({{.ArgCount}} 個) が多すぎます。\\n使用法: minikube config set PROPERTY_NAME PROPERTY_VALUE",
```
Comment (atoato88) [inline translations/ja.json:960]: Just comment.
s/toom any/too many/ right?

File: translations/ja.json [hunk 2]
```diff
@@ -954,64 +954,64 @@
...
+	"usage: minikube config unset PROPERTY_NAME": "使用法: minikube config unset PROPERTY_NAME",
+	"usage: minikube delete": "使用法: minikube delete",
+	"usage: minikube profile [MINIKUBE_PROFILE_NAME]": "使用法: minikube profile [MINIKUBE_PROFILE_NAME]",
+	"using metrics-server addon, heapster is deprecated": "metrics-server アドオンを使用します (heapster は廃止予定です)",
+	"version json failure": "JSON 形式のバージョン表示に失敗しました",
+	"version yaml failure": "YAML 形式のバージョン表示に失敗しました",
+	"zsh completion failed": "zsh のコマンド補完に失敗しました",
+	"zsh completion.": "zsh のコマンド補完です。",
+	"{{ .name }}: Suggestion: {{ .suggestion}}": "{{ .name }}: 提案: {{ .suggestion}}",
+	"{{ .name }}: {{ .rejection }}": "",
```
Comment (atoato88) [inline translations/ja.json:988]: No translations here?
Is this intended?

File: translations/ja.json [hunk 3]
```diff
@@ -954,64 +954,64 @@
...
-	"{{.driver}} only has {{.size}}MiB available, less than the required {{.req}}MiB for Kubernetes": "",
+	"{{.driver}} does not appear to be installed, but is specified by an existing profile. Please run 'minikube delete' or install {{.driver}}": "{{.driver}} がインストールされていないようですが、既存のプロフィールから指定されています。'minikube delete' を実行するか、{{.driver}} をインストールしてください",
+	"{{.driver}} only has {{.size}}MiB available, less than the required {{.req}}MiB for Kubernetes": "{{.driver}} は Kubernetes に必要な {{.req}}MiB 未満の {{.size}}MiB しか使用できません。",
 	"{{.err}}": "",
-	"{{.extra_option_component_name}}.{{.key}}={{.value}}": "{{.extra_option_component_name}}.{{.key}}={{.value}}",
-	"{{.name}} doesn't have images.": "",
-	"{{.name}} has following images:": "",
-	"{{.name}} has no available configuration options": "{{.name}} には利用可能なオプションがありません",
-	"{{.name}} is already running": "{{.name}} はすでに起動しています",
+	"{{.extra_option_component_name}}.{{.key}}={{.value}}": "",
```
Comment (atoato88) [inline translations/ja.json:1002]: Same comment before(at L988).

**Comment by atoato88** [comment]
I left some comments.
Could you check those? :smile:

File: translations/ja.json [hunk 4]
```diff
@@ -954,64 +954,64 @@
...
+	"{{.driver_name}} has less than 2 CPUs available, but Kubernetes requires at least 2 to be available": "{{.driver_name}} で利用できる CPU が 2 個未満ですが、Kubernetes を使用するには 2 個以上の CPU が必要です",
+	"{{.driver_name}} has only {{.container_limit}}MB memory but you specified {{.specified_memory}}MB": "{{.driver_name}} は {{.container_limit}}MB のメモリーしか使用できませんが、{{.specified_memory}}MB のメモリー使用を指定されました",
 	"{{.driver}} does not appear to be installed": "{{.driver}} がインストールされていないようです",
-	"{{.driver}} does not appear to be installed, but is specified by an existing profile. Please run 'minikube delete' or install {{.driver}}": "{{.driver}} がインストールされていないようですが、既存のプロフィールから指定されています。「 minikube delete 」を実行、あるいは {{.driver}} をインストールしてください",
-	"{{.driver}} only has {{.size}}MiB available, less than the required {{.req}}MiB for Kubernetes": "",
+	"{{.driver}} does not appear to be installed, but is specified by an existing profile. Please run 'minikube delete' or install {{.driver}}": "{{.driver}} がインストールされていないようですが、既存のプロフィールから指定されています。'minikube delete' を実行するか、{{.driver}} をインストールしてください",
```
Comment (t-inu) [inline translations/ja.json:999]:
```suggestion
	"{{.driver}} does not appear to be installed, but is specified by an existing profile. Please run 'minikube delete' or install {{.driver}}": "{{.driver}} がインストールされていないようですが、既存のプロファイルから指定されています。'minikube delete' を実行するか、{{.driver}} をインストールしてください",
```
他の和訳と合わせるため、「プロフィール」→「プロファイル」がよいです。

File: translations/ja.json [hunk 5]
```diff
@@ -954,64 +954,64 @@
...
+	"{{.driver}} only has {{.size}}MiB available, less than the required {{.req}}MiB for Kubernetes": "{{.driver}} は Kubernetes に必要な {{.req}}MiB 未満の {{.size}}MiB しか使用できません。",
```
Comment (t-inu) [inline translations/ja.json:1000]:
```suggestion
	"{{.driver}} only has {{.size}}MiB available, less than the required {{.req}}MiB for Kubernetes": "{{.driver}} は Kubernetes に必要な {{.req}}MiB 未満の {{.size}}MiB しか使用できません",
```
原文には「.」がないため「。」を削除。

File: translations/ja.json (hunk 1 above)
Comment (yosshy) [reply translations/ja.json:960]: @atoato88 right. It looks typo and I'll make an issues to fix it.

File: translations/ja.json (hunk 2 above)
Comment (yosshy) [reply translations/ja.json:988]: Yes. Looking at other translations, "" looks no translation (i.e. "{{ .name }}: {{ .rejection }}")

**Review by yosshy — COMMENTED** [review]
@atoato88 @t-inu Thank you for your reviews.

File: translations/ja.json (hunk 2 above)
Comment (atoato88) [reply translations/ja.json:988]: If no translations (i.e. `"xxxxxxxx": "",` ), minikube use original text "xxxxxxxx" as is, right?

**Comment by atoato88** [comment]
/lgtm
OK as Japanese review.

**Comment by yosshy** [comment]
/assign @sharifelgamal
It’s ready to merge. Would you merge it?

**Review by sharifelgamal — APPROVED** [review]

**Comment by yosshy** [comment]
@sharifelgamal thanks for merging it.

**Pull request merged**